frontend/src/components/AdminSettings.vue

```diff
@@ -229,6 +229,20 @@ const betaUpdateExists = computed(() => {
   return false;
 });
 
+const remainingSeats = computed(() => {
+  if (admin.value) {
+    return admin.value.licensedSeats - admin.value.usedSeats;
+  }
```

Comment:
What is the value, if we are not admin?

Reply:
`null`, the billing resource is protected by the admin role.